examples/platform/silabs/SiWx917/SiWx917/wfx_rsi.h

```diff
@@ -21,10 +21,10 @@
  * Interface to RSI Sapis
  */
 
-#define WFX_RSI_WLAN_TASK_SZ (1024 + 512 + 256 + 1024 + 512) /* Unknown how big this should be 	*/
-#define WFX_RSI_TASK_SZ (1024 + 1024 + 1024)                 /* Stack for the WFX/RSI task		*/
-#define WFX_RSI_BUF_SZ (1024 * 15)                           /* May need tweak 			*/
-#define WFX_RSI_CONFIG_MAX_JOIN (5)                          /* Max join retries			*/
+#define WFX_RSI_WLAN_TASK_SZ (1024 + 512 + 256) /* Unknown how big this should be 	*/
```

Comment:
Does the comment still apply? Seems like you did the calculation on how big it should be.

Reply:
Yes this comment should not apply now.